.env.test

```diff
@@ -0,0 +1,16 @@
+DEBUG=true
+SECRET_KEY='tests'
+DATABASE_URL='postgresql:///opencraft'
```

Comment:
Can we use a different name for the test database, `opencraft-test` maybe? I'd prefer to reserve the name `opencraft` for my development database.